cloudfoundry-example/build.gradle

```diff
@@ -0,0 +1,28 @@
+plugins {
+	id 'java'
+	id 'application'
+	id "com.github.johnrengelman.shadow" version "2.0.2"
```

Comment:
Did you there's a Vert.x Gradle community plugin? https://github.com/jponge/vertx-gradle-plugin

Reply:
Yes. I've even started contributing to that :)